partner_contact_company/models/partner.py

```diff
@@ -7,41 +7,47 @@
 
 
 class ResPartner(models.Model):
+    """Enhance the feature of Partner."""
```

Comment:
Can you remove this doc, I think that inheriting existing model do not need any extra documentation. It already explicite, and adding such extra documentation mean also maintaining it.